zipline/utils/calendars/trading_calendar.py

```diff
@@ -95,6 +96,14 @@ def __init__(self, start=start_default, end=end_default):
             dtype='datetime64[ns]',
         )
 
+        # Simple cache to avoid recalculating the same minute -> session in
+        # "next" mode. Analysis of current zipline code paths show that
+        # `minute_to_session_label` is often called consecutively with the same
+        # inputs.
+        self._minute_to_session_label_cache = LRU(1)
+
+        self._session_nanos = self.schedule.index.values.astype(np.int64)
```

Comment:
When I import zipline ,it returns  'ImportError: No module named 'lru''. 
My system is win10-64bit,and environment is python 3.5. Would you please tell me what should I do to fix it?